packages/mobile/src/components/overflow-menu-drawer/CollectionOverflowMenuDrawer.tsx

```diff
@@ -109,7 +111,10 @@ const CollectionOverflowMenuDrawer = ({ render }: Props) => {
     [OverflowAction.DELETE_PLAYLIST]: () =>
       dispatch(openDeletePlaylist({ playlistId: id })),
     [OverflowAction.PUBLISH_PLAYLIST]: () =>
-      onOpen({ contentId: Number(id), contentType: 'playlist' })
+      openPublishConfirmation({
+        contentType: is_album ? 'album' : 'playlist',
+        confirmCallback: dispatch(publishPlaylist(Number(id)))
```

Comment:
don't we need to wrap this in a function? otherwise it'll dispatch `publishPlaylist` as this open call is executed?

Reply:
ohhhh good catch.